tests/viewset_test.py

```diff
@@ -0,0 +1,66 @@
+from unittest import mock
+
+import pytest
+from rest_framework import status
+from rest_framework.reverse import reverse
+from rest_framework.test import APIClient
+
+
+def test_posting_valid_data_returns_utm_zones(api_client, utm_zone_post_url, payload):
+    expected_srid = 123456
+    utm_zone_mock = mock.Mock()
+    utm_zone_mock.srid = expected_srid
+    with mock.patch(
+            'utm_zone_info.viewsets.utm_zones_for_representing', return_value=[utm_zone_mock]):
+        post_viewset_result = api_client.post(utm_zone_post_url, payload, format='json')
+        expected_result = {'utm_zone_srids': [expected_srid]}
+        assert post_viewset_result.status_code == status.HTTP_200_OK
+        assert post_viewset_result.json() == expected_result
```

Comment:
Consider to also assert whether the (mocked) `utm_zones_for_representing` was called with an argument reflecting the playload (incl. SRID).